### DISCUSSION TITLE
xopp file open

### DISCUSSION THREAD

**Discussion by Yingfeng-Zhac** [xournalpp/xournalpp#4735] [Q&A]
I use a mac M2 with xournal++ 1.1.3. I can't  open the .xopp file with double-click, I have to use File>open to open it. How to fix it
![Snipaste_2023-03-16_15-56-02](https://user-images.githubusercontent.com/128027566/225551069-b4bb5f58-2f20-4b11-a77f-2616fb064f68.png)

**Comment by rolandlo** [comment]
It's currently not possible to open `.xopp` files from Finder on MacOS.
In general I would advise you to use the [nightly build](https://github.com/xournalpp/xournalpp/releases/tag/nightly) on MacOS for a better experience.

**Reply by Yingfeng-Zhac** [reply]
Thanks for your replay. I tried the nightly build, It's better for MacOS. Thanks again!
